integrations/access/pagerduty/app.go

```diff
@@ -113,7 +126,10 @@ func (a *App) run(ctx context.Context) error {
 	watcherJob, err := watcherjob.NewJob(
 		a.teleport,
 		watcherjob.Config{
-			Watch:            types.Watch{Kinds: []types.WatchKind{{Kind: types.KindAccessRequest}}},
+			Watch: types.Watch{Kinds: []types.WatchKind{
+				{Kind: types.KindAccessRequest},
+				{Kind: types.KindAccessMonitoringRule},
```

Comment:
Will this have the same issue as we fixed for Slack if the plugin does not have permissions to watch KindAccessMonitoringRule resources?